app/controllers/refinery/teams/admin/categories_controller.rb

```diff
@@ -43,8 +38,18 @@ def find_category
 
         def after_update_positions
           find_all_categories
-          render :partial => '/refinery/teams/admin/categories/sortable_list' and return
+          render partial: '/refinery/teams/admin/categories/sortable_list' and return
```

Comment:
Line is too long. [85/80]<br>Use `&&` instead of `and`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.